config/default.yaml

```diff
@@ -73,9 +73,12 @@ network:
     rinkeby:
       type: "eth"
       address: "https://rinkeby.infura.io/v3/400b1ad4183b442dbd9b0b5744b327aa" # TODO: project ID
-#    solana_devnet:
-#      type: "solana"
-#      address: "https://api.devnet.solana.com"
+    # solana_devnet:
+    #   type: "solana"
+    #   address: "https://api.devnet.solana.com"
+    # solana:
```

Comment:
I added Solana mainnnet details here (commented out) because it's much easier to test domain names in mainnet.